frontend/packages/kubevirt-plugin/integration-tests-cypress/tests/tabs/details-resources-requirement.spec.ts

```diff
@@ -0,0 +1,105 @@
+import { testName } from '../../support';
+import { K8S_KIND, VM_STATUS, YAML_VM_NAME } from '../../utils/const/index';
+import {
+  detailsTab,
+  modalConfirm,
+  modalTitle,
+  pendingChangeAlert,
+  saveAndRestart,
+} from '../../views/selector';
+import { tab } from '../../views/tab';
+import { waitForStatus, vm } from '../../views/vm';
+
+const flavor = '[data-test-id="details-Flavor"]';
```

Comment:
Is there a reason this selector and the ones below aren't in a selectors or view file?

Reply:
I put these selectors/strings inside the test file on purpose because there is very little chance to reuse them as they're dedicated to a specific tab. If we find out it's not the case in future, then let's putting them in a view file. Does it sound good?

Reply:
I think it makes more sense to default to putting selectors in a dedicated view file because when I go to write a test, I assume that all items related to a given view are located there. It provides a single source of truth for views and avoids potential duplication with no down side.

Reply:
Sounds good!